src/StickerService.ts

```diff
@@ -50,6 +53,48 @@ function buildWallpaperCss({
     background-size: cover !important;
   }
 
+  /* Output panel */
+  .monaco-workbench .part.panel > .content .monaco-editor .monaco-editor-background,
+  .margin-view-overlays, 
+  .margin, 
+  /* Welcome screen */
+  .editor-instance,
+  .welcomePageContainer,
+  .welcomePage,
+  .welcomePageFocusElement
+  {
+    background-color: transparent !important;
+  }
+
+  /* Glass notifications */
+  .notification-toast 
+  {
+    backdrop-filter: blur(3px) !important;
+    background-color: rgba(0, 255, 0, 0.4) !important;
+  }
+
+  .monaco-list,
```

Comment:
```
.monaco-list,
  .monaco-list-rows,
  .monaco-list-row
```

So these selectors are all over the place. This could potentially affect a lot of other things. I'd like to narrow things down to where it's actually needed. That way something obscure isn't happening somewhere else :)

So what so what where you trying to style with this?

Reply:
Maybe we could just apply it to all of the child list items of a targeted parent (such as the command palette thing)

https://stackoverflow.com/questions/26349987/how-do-i-apply-a-style-to-all-children-of-an-element

That way you get all of them where they are need. (I think that should work)

Does that make sense?